tools/state-viewer/src/cli.rs

```diff
@@ -70,6 +70,9 @@ pub enum StateViewerSubCommand {
     /// View trie structure.
     #[clap(alias = "view_trie")]
     ViewTrie(ViewTrieCmd),
+    /// View the best window for maintenance in current epoch
+    #[clap(alias = "maintenance_window")]
```

Comment:
The other aliases are only for backwards compatibility.  New commands shouldn’t have the alias and thus should use the default dashes.

```suggestion
```

Reply:
Got it, thanks, man.